ClientFactory/PluginClientFactory.php

```diff
@@ -2,8 +2,8 @@
 
 namespace Http\HttplugBundle\ClientFactory;
 
-use Http\Client\Plugin\Plugin;
-use Http\Client\Plugin\PluginClient;
+use Http\Client\Common\Plugin;
+use Http\Client\Common\PluginClient;
```

Comment:
i think this is technically a BC break - if somebody typehinted the old plugin client, things will fail. but i think thats really an edge case, just mention it in the changelog.